clients/hadoopfs/pom.xml

```
@@ -256,6 +256,13 @@ To export to S3:
             <version>3.12.0</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.spark</groupId>
+            <artifactId>spark-sql_2.12</artifactId>
+            <version>3.0.1</version>
+            <scope>provided</scope>
```

I don't think that this code changes the client in any way, only the tests.  So I believe that the client still does not have a Spark dependency -- only the test.  Can we instead try this?

```suggestion
            <scope>test</scope>
```